modules/RTC/TraceablePeerConnection.js

```diff
@@ -1495,6 +1543,42 @@ TraceablePeerConnection.prototype.setAudioTransferActive = function(active) {
     return changed;
 };
 
+/**
+ * Takes in a *unified plan* offer and inserts the appropriate
+ * parameters for adding simulcast receive support.
+ * @param {} A session description object (with 'type' and 'sdp' fields)
+ * @return {} A session description object with its sdp field modified to
+ * advertise simulcast receive support
```

Comment:
Does the passed in description get modified in place? Is the return necessary then?